core-js/src/main/javascript/cdf/addIns/template/templateBase.js

```diff
@@ -108,7 +109,7 @@ define([
           switch(opt.templateType.toUpperCase()) {
             case 'UNDERSCORE':
               model = _.defaults({}, model, Utils.propertiesArrayToObject(helpers));
-              html = _.template(Utils.ev(opt.template))(model);
+              html = _.template((_.isFunction(opt.template) ? opt.template() : opt.template), model);
```

Comment:
[Utils.ev](https://github.com/webdetails/cdf/blob/master/core-js/src/main/javascript/cdf/dashboard/Utils.js#L375) does this. Again, due to the change to _.template, this change will break